src/languages/en.ts

```diff
@@ -313,7 +313,7 @@ const translations = {
         addressLine: ({lineNumber}: AddressLineParams) => `Address line ${lineNumber}`,
         personalAddress: 'Personal address',
         companyAddress: 'Company address',
-        noPO: 'PO boxes and mail drop addresses are not allowed',
+        noPO: 'No PO boxes or mail-drop addresses, please.',
```

Comment:
This will need to be updated in the `es.ts` file as well

Reply:
All good? Thank you!